fvm/transactionInvoker.go

```diff
@@ -80,30 +81,34 @@ func (i *TransactionInvoker) Process(
 
 	env := NewTransactionEnvironment(*ctx, vm, sth, programs, proc.Transaction, proc.TxIndex, span)
 
-	predeclaredValues := valueDeclarations(env)
-
 	location := common.TransactionLocation(proc.ID)
 
+	runtimeEnv := runtime.NewBaseInterpreterEnvironment(runtime.Config{})
+	declareStandardLibraryValues(runtimeEnv, env)
```

Comment:
A great optimization opportunity would be to reuse this environment across multiple transactions (and also for the contract invocations). Likewise, for script executions we should create and reuse a single script environment.

Reply:
cc: @pattyshack

Reply:
creating a new env is relatively cheap after moving out the meter parameters stuff.  are we reusing the standard lib values on cadence side (cuz re-creating those are expensive if i recall correctly)?